graphdb/config/docker-compose-start.yml

```diff
@@ -0,0 +1,24 @@
+version: "3"
+
+services:
+  graphdb:
+    container_name: ${GRAPHDB_CONTAINER_NAME}
+    image: ontotext/graphdb:${GRAPHDB_VERSION}-free
```

Comment:
There's no such image in the docker hub. You should update the readme to include info how to build an image with the free version and set it here